src/index.ts

```diff
@@ -60,3 +60,4 @@ export type {
 	RespondProxy,
 } from './messenger';
 export { postMessage } from './messenger/post-message';
+export { buildPageTargeting } from './targeting/build-page-targeting';
```

Comment:
Not certain but it might be handy to export the `PageTargeting` type to consumers of commercial-core as well.

Reply:
Good idea, I've added this. Will update consumers in the upcoming frontend PR